app/Console/Commands/CacheMiturAbruzzoApiCommand.php

```diff
@@ -510,10 +510,26 @@ protected function getImagesFromOsmfeaturesData($osmfeaturesData)
         if (!isset($osmfeaturesData['images'])) {
             return $images;
         }
-        foreach ($osmfeaturesData['images'] as $image) {
-            // add only $image['source_url'] with extension jpg, jpeg, png, bmp, gif, webp, svg (to avoid other files)
-            if (in_array(pathinfo($image['source_url'], PATHINFO_EXTENSION), ['jpg', 'jpeg', 'png', 'bmp', 'gif', 'webp', 'svg'])) {
-                $images[] = $image['source_url'];
+
+        $imageSections = ['wikipedia_images', 'wikidata_images', 'wikimedia_images'];
```

Comment:
quando abbiamo a che fare con etichette conviene sempre usare enumerazioni, il motivo è semplice permette di divincolarci dal vero valore che puo cambiare nel tempo esempio se noi avessimo  un enum $wikiImageType e alla etichetta "wikipedia" assegnassimo "wikipedia_images" un domani se questa etichetta cambia non dobbiamo cercare in tutto il codice ma modificare solo l'enum, inoltre l'enum ci garantisce un controllo in piu che elimina l'errore umano se non scrivo correttamente il nome dell'attributo segnala errore

Reply:
Cambiamento apportato